docs/userguide/networking/get-started-overlay.md

```diff
@@ -335,7 +335,7 @@ You can try starting a second network on your existing Swarm cluster using Docke
 
 6. Start the application with Compose.
 
-		$ docker-compose --x-networking --project-name=counter up -d
+		$ docker-compose --x-networking up -d
```

Comment:
Looks like the `--project-name` got lost here (added in https://github.com/docker/docker/commit/f16dceb63233bd2c1fd327b92d7112e8012347ac), the name is required for the example, because the `MONGO_HOST` env var assumes a specific name for the container